src/docs/development/platform-integration/platform-views.md

```diff
@@ -212,27 +260,28 @@ For more information, see the API docs for:
 [`PlatformViewRegistry`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformViewRegistry.html
 [`PlatformView`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformView.html
 
-Finally, modify your `build.gradle` file to require
-version 19 (or later) of the Android API.
+Finally, modify your `build.gradle` file to require either 
+the minimal Android SDK version:
 
 ```gradle
 android {
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 19 // if using Hybrid composition.
+        minSdkVersion 20 // if using Virtual display.
     }
 }
 ```
 
 ### Performance
 
 Hybrid composition on Android works best on Android 10 or above.
```

Comment:
I'll add a section about VD performance, maybe something like:

Each pixel is flowing through additional intermediate graphic buffers (that are as big as the platform view) which cost in graphic memory and drawing perfomance.

Reply:
Can we say something about the performance post Android 10? I'm assuming we have similar performance hit to VDs?